setup.cfg

```diff
@@ -4,26 +4,27 @@ universal = 1
 [tool:pytest]
 addopts = -v --cov=open_alchemy --cov=examples/app --cov-config=tests/.coveragerc --flake8 --strict --cov-report xml --cov-report term
 markers =
-    model
-    column
-    integration
-    helper
-    app
-    utility_base
-    table_args
-    facade
-    sqlalchemy
-    models_file
-    only_this
-    example
-    init
-    slow
-    schemas
-    code_formatter
-    mixins
-    artifacts
-    validation
-    build
+  app
```

Comment:
I'm interested in this, did you manually re-order it?

Reply:
Nope, I used https://www.online-utility.org/text/sort.jsp.

Reply:
Ah nice, might be splitting hairs, another part of this file uses a tab width of 4 😄